static/js/search.js

```diff
@@ -2,9 +2,33 @@ var search = (function () {
 
 var exports = {};
 
+function ValidURL(str)
+{
+    var patt = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!-\/]))?/;
+    if (!patt.test(str)) {
+        return false;
+    } else {
+        return true;
+    }
```

Comment:
I think the if/else statement here can be written a bit more simply as `return patt.test(str)`